crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI044.pyi

```diff
@@ -1,5 +1,6 @@
 # Bad import.
 from __future__ import annotations # PYI044.
+from __future__ import annotations, OtherThing # PYI044.
```

Comment:
nit: let's use another import that actually exists in the `__future__` module ;)

```suggestion
from __future__ import annotations, with_statement # PYI044.
```